lightwood/data/timeseries_transform.py

```diff
@@ -162,7 +162,7 @@ def transform_timeseries(
         if tss.allow_incomplete_history:
             log.warning("Forecasting with incomplete historical context, predictions might be subpar")
         else:
-            raise Exception(f'Not enough historical context to make a timeseries prediction. Please provide a number of rows greater or equal to the window size. If you can\'t get enough rows, consider lowering your window size. If you want to force timeseries predictions lacking historical context please set the `allow_incomplete_history` timeseries setting to `True`, but this might lead to subpar predictions.') # noqa
+            raise Exception(f'Not enough historical context to make a timeseries prediction ({min(group_lengths)} < {tss.window}). Please provide a number of rows greater or equal to the window size. If you can\'t get enough rows, consider lowering your window size. If you want to force timeseries predictions lacking historical context please set the `allow_incomplete_history` timeseries setting to `True`, but this might lead to subpar predictions.') # noqa
```

Comment:
When this outputs does it show the numbers? If so move as follows:
"
'Not enough historical context to make a timeseries prediction. Please provide a number of rows greater or equal to the window size - currently number of rows to window size = ({min(group_lengths)} < {tss.window}). If you can\'t get enough rows, consider lowering your window size. If you want to force timeseries predictions lacking historical context please set the `allow_incomplete_history` timeseries setting to `True`, but this might lead to subpar predictions.')
"

Reply:
real silly nitpick but it's just to make sense of the numbers that will just automatically pop up.